python/cugraph/cugraph/sampling/node2vec.py

```diff
@@ -115,24 +124,22 @@ def node2vec(G,
     dsts = G.edgelist.edgelist_df['dst']
     weights = G.edgelist.edgelist_df['weights']
 
-    resource_handle = pylibcugraph.experimental.ResourceHandle()
-    graph_props = pylibcugraph.experimental.GraphProperties(
-                    is_multigraph=G.is_multigraph())
+    resource_handle = ResourceHandle()
+    graph_props = GraphProperties(is_multigraph=G.is_multigraph())
     store_transposed = False
     renumber = False
     do_expensive_check = False
 
     # FIXME: If input graph is not renumbered, then SGGraph creation
     # causes incorrect vertices to be returned when computing pylib
     # version of node2vec
-    sg = pylibcugraph.experimental.SGGraph(resource_handle, graph_props,
-                                           srcs, dsts, weights,
-                                           store_transposed, renumber,
-                                           do_expensive_check)
-
-    vertex_set, edge_set, sizes = pylibcugraph.experimental.node2vec(
-                                    resource_handle, sg, start_vertices,
-                                    max_depth, use_padding, p, q)
+    sg = SGGraph(resource_handle, graph_props, srcs, dsts, weights,
+                 store_transposed, renumber, do_expensive_check)
+
+    vertex_set, edge_set, sizes = \
```

Comment:
I am seeing you are not doing the conversion when `compress_result=False` (like you did in your previous PR when `use_padding=True`). Is that taken care of the the C++ layer when passing the parameter `compress_result`?

Reply:
Got it. Thanks